resolwe/flow/models/base.py

```diff
@@ -18,6 +20,40 @@
 MAX_SLUG_LENGTH = 100
 MAX_NAME_LENGTH = 100
 
+# Data status enum.
+
+
+class DataStatus(StrEnum):
+    #: data object is uploading
+    UPLOADING = "UP"
+    #: data object is being resolved
+    RESOLVING = "RE"
+    #: data object is waiting
+    WAITING = "WT"
+    #: data object is preparing
+    PREPARING = "PP"
+    #: data object is processing
+    PROCESSING = "PR"
+    #: data object is done
+    DONE = "OK"
+    #: data object is in error state
+    ERROR = "ER"
+    #: data object is in dirty state
+    DIRTY = "DR"
+
+    @classmethod
+    def collection_status_order(cls):
```

Comment:
This is not only for collections but also for entities. And, it could also be used to sort data objects themselves.

Reply:
I agree... Should I rename it to the `status_sorting_order` ?

Reply:
It is already in `DataStatus` class, so I would prefer just `sort_order`.

Reply:
OK.